_data-prepper/trace-analytics.md

```diff
@@ -8,40 +8,44 @@ nav_order: 21
+The following flowchart illustrates the trace analytics workflow, from running OpenTelemetry Collector to using OpenSearch Dashboards for visualization.
 
-![Trace analytics component]({{site.url}}{{site.baseurl}}/images/data-prepper/trace-analytics/trace-analytics-components.png)
+<img src="{{site.url}}{{site.baseurl}}/images/data-prepper/trace-analytics/trace-analytics-components.png" alt="Trace analyticis component overview">{: .img-fluid}
 
-In your service environment, you need to run OpenTelemetry collector. You can run it as a sidecar or daemonset for EKS, a sidecar for ECS, or an agent on EC2. You should configure the collector to export trace data to Data Prepper. Then, you need to deploy Data Prepper as an intermediate component and configure it to send the enriched trace data to your OpenSearch cluster or Amazon OpenSearch Service domain. Finally, use OpenSearch Dashboards to visualize and detect problems in your distributed applications.
+To monitor trace analytics, you need to set up the following components in your service environment:
+- Add an **instrumentation** to your application so it can generate telemetry data and send it to an OpenTelemetry collector.
+- Run an **OpenTelemetry collector** as a sidecar or daemonset for EKS, a sidecar for ECS, or an agent on EC2. You should configure the collector to export trace data to Data Prepper. 
+- Deploy **Data Prepper** as an intermediate component and configure it to send the enriched trace data to your OpenSearch cluster or Amazon OpenSearch Service domain. 
+- Use **OpenSearch Dashboards** to visualize and detect problems in your distributed applications.
 
 ## Trace analytics pipeline
 
 To achieve trace analytics in Data Prepper, we have three pipelines: `entry-pipeline`, `raw-trace-pipeline` and `service-map-pipeline`.
 
-![Trace analytics pipeline]({{site.url}}{{site.baseurl}}/images/data-prepper/trace-analytics/trace-analytics-feature.jpeg)
+<img src="{{site.url}}{{site.baseurl}}/images/data-prepper/trace-analytics/trace-analytics-feature.jpg" alt="Trace analyticis pipeline overview">{: .img-fluid}
 
 
 ### OpenTelemetry trace source
 
-The [OpenTelemetry source](../data-prepper-plugins/otel-trace-source/README.md) accepts trace data from the OpenTelemetry collector. The source depends on [OpenTelemetry Protocol](https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/protocol). The source officially support transport over gRPC. The source also supports industry-standard encryption (TLS/HTTPS).
+The [OpenTelemetry source]({{site.url}}{{site.baseurl}}/data-prepper-plugins/otel-trace-source/README.md) accepts trace data from the OpenTelemetry collector. The source follows the [OpenTelemetry Protocol](https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/protocol). The source officially supports transport over gRPC and the use of industry-standard encryption (TLS/HTTPS).
```

Comment:
> data-prepper-plugins/otel-trace-source/README.md

This is not the correct URL.

This should link to the path which is currently generated as: `/docs/2.4/data-prepper/configuration/sources/otel-trace/`. I'm not sure quite what `site.baseurl` resolves to, so you might need to change the path some.